apps/backend/src/statusActionEngine/statusActionUtils.ts

```diff
@@ -99,6 +155,51 @@ export const getEmailReadyArrayOfUsersAndProposals = async (
           ? await usersDataSource.getProposalUsers(proposal.primaryKey)
           : null;
 
+        const callDataSource = container.resolve<CallDataSource>(
+          Tokens.CallDataSource
+        );
+        let techniques: Technique[] = [];
+        let hazardAnswers: Answer[] = [];
+        let sampleAnswers: Answer[] = [];
+        const XCalls = await callDataSource
+          .getCalls({
+            proposalStatusShortCode: 'QUICK_REVIEW',
+          })
+          .then((calls) => calls.map((call) => call.id));
+        if (XCalls.includes(proposal.callId)) {
+          const techniqueDataSource: TechniqueDataSource = container.resolve(
+            Tokens.TechniqueDataSource
+          );
+          techniques = await techniqueDataSource.getTechniquesByProposalPk(
+            proposal.primaryKey
+          );
+          const questionaryDataSource: QuestionaryDataSource =
+            container.resolve(Tokens.QuestionaryDataSource);
+          const questionarySteps =
+            await questionaryDataSource.getQuestionarySteps(
+              proposal.questionaryId
+            );
+          for await (const step of questionarySteps) {
+            const stepFields = step.fields.map((field) => field);
+            if (step.topic.title === 'Samples') {
+              const answers = await stepAnswers(
+                stepFields,
+                proposal.primaryKey
+              );
+              if (answers !== null) {
+                sampleAnswers = sampleAnswers.concat(answers);
+              }
+            } else if (step.topic.title === 'Hazards') {
+              const answers = await stepAnswers(
+                stepFields,
+                proposal.primaryKey
+              );
+              if (answers !== null) {
```

Comment:
shall we also check for answers list size 0; it is unnecessary to concat a blank list. what do you say?

Reply:
Updated